packages/rs-drive-abci/Cargo.toml

```diff
@@ -74,6 +72,8 @@ tokio = { version = "1", features = [
 ] }
 tokio-util = { version = "0.7.8" }
 derive_more = "0.99.17"
+metrics = "0.22.1"
```

Comment:
Please revert unrelated changes

Reply:
this change is related: previous version of `metrics` indirectly had version of another library that went in conflict with one used in faker dependency

Reply:
nvm it seems dev was updated in these months